extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java

```diff
@@ -145,7 +145,8 @@ public static JavaSettings getInstance() {
                 getBooleanValue(host, "use-default-http-status-code-to-exception-type-mapping", false),
                 host.getValue(new TypeReference<Map<Integer, String>>() {}.getType(),
                     "http-status-code-to-exception-type-mapping"),
-                getBooleanValue(host, "partial-update", false)
+                getBooleanValue(host, "partial-update", false),
+                getBooleanValue(host, "no-named-response-types", false)
```

Comment:
Emm, possible for a better name? `generic-response-type`?
Word `no` seems a bit strange to me.

Off-topic, we might want to maintain some guide on collection of recommended options for vanilla SDK. Even I do not know what best to choose if I want to make an vanilla SDK.

Reply:
Great. It is not urgent and not even need to be very detail. It would be already great if it works a reminder for us.

Just a note, I put the default flags in DPG at its readme (low-level-client).
https://github.com/Azure/autorest.java/blob/main/preprocessor/readme.md

And made a simple doc for potential service team working on it.
https://github.com/Azure/autorest.java/blob/main/docs/generate/dataplane.md

Of course vanilla would likely take a different format for all these.